src/ditto/fx/passes/parallelize_linear.py

```diff
@@ -51,16 +51,22 @@ def call(self, graph_module: GraphModule) -> PassResult:
         graph = graph_module.graph
         lm_head = Linear.find_last(graph)
         for node in graph.nodes:
-            if not ((fused_linear := FusedLinear.configure_from(node)) or (linear := Linear.configure_from(node))):
+            if not (
+                (
+                    (fused_linear := FusedLinear.configure_from(node))
+                    or (linear := fused_linear.linear if fused_linear else Linear.configure_from(node))
+                )
+                and linear is not None
+            ):
```

Comment:
When `fused_linear is not None`, the expression `(linear := fused_linear.linear if fused_linear else Linear.configure_from(node))` won't be evaluated, making `linear` an undefined identifier in the subsequent flow.
```suggestion
            if not (
                linear := (
                    fused_linear.linear
                    if (fused_linear := FusedLinear.configure_from(node))
                    else Linear.configure_from(node)
                )
            ):
```